var/spack/repos/builtin/packages/mrbayes/package.py

```diff
@@ -15,11 +15,14 @@ class Mrbayes(AutotoolsPackage):
     homepage = "http://mrbayes.sourceforge.net"
     git      = "https://github.com/NBISweden/MrBayes.git"
 
-    version('2017-11-22', commit='8a9adb11bcc538cb95d91d57568dff383f924503')
+    version('3.2.7a', commit='0176ac2d0bfed53a5bc0aaf3f5a3def71f23575f')
```

Comment:
If we use a tarball instead of a git checkout, we might be able to get rid of those Autotools dependencies.

Reply:
Thanks for pointing out. I have substituted git checkouts for tarballs, therefore deleting the Autotools dependencies.